src/ipc-handlers.ts

```diff
@@ -1711,27 +1712,61 @@ export function comparePaths( event: IpcMainInvokeEvent, path1: string, path2: s
 	return arePathsEqual( path1, path2 );
 }
 
-export async function listWpContentFolders(
+export async function listLocalFileTree(
 	_event: Electron.IpcMainInvokeEvent,
 	siteId: string,
-	subdir: GranularSyncFolders
-): Promise< { name: string; type: 'file' | 'folder' }[] > {
+	path: string = 'wp-content',
```

Comment:
```suggestion
	path: string,
```
Nit, but it seems reasonable to require a path argument for this function.

Reply:
https://github.com/Automattic/studio/pull/1940/commits/30c8bc1d8bcedb6aa276d950c32d59bf4f848599